.github/workflows/lsst-tests.yml

```diff
@@ -60,3 +59,19 @@ jobs:
         uses: codecov/codecov-action@v3
```

Comment:
You are going to want to update your codecov action to v4 and include the per-repo token (see daf_butler).